server/src/entity/Excuse.ts

```diff
@@ -0,0 +1,47 @@
+import { ID, ObjectType, Field } from 'type-graphql'
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity, ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn
+} from 'typeorm'
+import { Student } from './Student'
+import { User } from './User'
+
+@Entity()
+@ObjectType()
+export class Excuse extends BaseEntity {
+    @PrimaryGeneratedColumn('uuid')
+    @Field(() => ID)
+    id: string
+
+    @CreateDateColumn()
+    @Field()
+    createdAt: number
+
+    @UpdateDateColumn()
+    @Field()
+    updatedAt: number
+
+    @Column('date')
+    @Field()
+    startDate: string
+
+    @Column('date')
+    @Field()
+    endDate: string
+
+    @Column({ nullable: true })
+    @Field()
+    lessons?: string
```

Comment:
Ich habe gestern nicht dran gedacht, aber kannst du mal schauen ob du hier einen Transformer implementieren kannst? Die Doku dazu ist hier der letzte Punkt in der List https://typeorm.io/#/entities/column-options. Damit könnten wir Excuse.lessons direkt als Int-Array deklarieren und in der Datenbank trotzdem als string abspeichern. Das würde die Konvertierungen und das Parsen anderswo vermeiden.

Reply:
angepasst :)